hudi-timeline-service/pom.xml

```diff
@@ -117,6 +123,28 @@
       <artifactId>rocksdbjni</artifactId>
     </dependency>
 
+    <!-- Jetty -->
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <version>${jetty.version}</version>
+    </dependency>
```

Comment:
How was this module getting the jetty dependencies earlier ? Was it happening transitively ?

Also no need to mention `jetty.version` while defining again.

Reply:
+1

Reply:
I believe it was getting these dependencies transitively from the `javalin` that we define as a compile time dependency in the `timeline-service` pom. https://github.com/apache/hudi/blob/master/hudi-timeline-service/pom.xml#L113 

The reason why i defined them here explicitly was because I ran into issues with tests failing due to getting incorrect jetty (9.3 jetty) from hadoop hive side, so i followed a similar idea as to what hudi-flink was doing by defining them within the pom https://github.com/apache/hudi/blob/master/packaging/hudi-flink-bundle/pom.xml#L612 (note hudi flink bundle is also shading them which also might be nescarry in our case)